Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11.5
+FROM golang:1.11.6
```

Comment:
Actually, looks like docker image 1.11.6 not yet available. So if doing release today, will have to keep 1.11.5

Reply:
Will keep checking on avail.. may be coming today?

Reply:
idk

Reply:
avail now, so merging.